ddtrace/contrib/wsgi/wsgi.py

```diff
@@ -80,73 +81,107 @@ def default_wsgi_span_modifier(span, environ):
     span.resource = "{} {}".format(environ["REQUEST_METHOD"], environ["PATH_INFO"])
 
 
-class DDWSGIMiddleware(object):
-    """WSGI middleware providing tracing around an application.
-
-    :param application: The WSGI application to apply the middleware to.
-    :param tracer: Tracer instance to use the middleware with. Defaults to the global tracer.
-    :param span_modifier: Span modifier that can add tags to the root span.
-                            Defaults to using the request method and url in the resource.
-    """
-
-    def __init__(self, application, tracer=None, span_modifier=default_wsgi_span_modifier):
+class DDWSGIMiddlewareBase(object):
```

Comment:
Can you explain why this additional base class is added?

Reply:
I updated the PR description